python/CCLE_data.py

```diff
@@ -11,43 +11,285 @@
 which can be used for the histone correlation calculations
 """
 
-import GEOparse
 import mygene
+import pandas as pd
+import numpy as np
+from sklearn import preprocessing
+import scipy.io as spio
+import os
+from scipy.stats.stats import pearsonr
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+
+
+
 
 """
-GETTING GENE NAMES
+FUNCTIONS
 """
+#Used to find common celllines
+def common_celllines(a,b):
```

Comment:
Best software developement practice suggests having docstrings instead of comments for describing functions: https://realpython.com/documenting-python-code/

Make docstrings for all functions.